### ISSUE TITLE
Typescript import helper messed up, project-specific

### ISSUE THREAD

**Issue by flq** [Microsoft/vscode#55992]
```
Version: 1.25.1
Commit: 1dfc5e557209371715f655691b1235b6b26a06be
Date: 2018-07-11T15:43:53.668Z
Electron: 1.7.12
Chrome: 58.0.3029.110
Node.js: 7.9.0
V8: 5.8.283.38
Architecture: x64
```
![image](https://user-images.githubusercontent.com/123748/43827870-38264432-9afb-11e8-868b-87b88f6620c4.png)

Steps to Reproduce:

![error capture](https://user-images.githubusercontent.com/123748/43827884-46badc1a-9afb-11e8-891e-de94d6234471.gif)

Does this issue occur when all extensions are disabled?: **Yes**

> I know that this is not exactly a VS Code issue, because in a different project I don't have this problem, but I am wondering whether this has been previously observed or as to what can confuse the importer. Everything else (type introspection, error highlighting, etc.) seems to work.

**Comment by ramya-rao-a** [comment]
Does it happen in the [latest Insiders](https://code.visualstudio.com/insiders) as well?

**Comment by mjbvz** [comment]
Should be fixed in 1.26. Closing since we have not heard otherwise
